src/main/scala/com/stratio/hermes/helpers/RandomHelper.scala

```diff
@@ -24,10 +24,27 @@ import scala.util.Random
 object RandomHelper {
 
   /**
-   * Returns a random element from a list.
-   * @param l initial list
-   * @tparam T with the type of the list
-   * @return a random element of the list or None if the list is empty.
-   */
+    * Returns a random element from a list.
```

Comment:
Indentation is wrong in comments. Configure your intellij. The same in the rest of files of your PR.